tools/resocsec

```diff
@@ -233,6 +254,10 @@ def resocsec():
 
     if 'func' in args:
         args.func(args)
+        if (args.subparser_name == 'make_secure_bl1_image' and
```

Comment:
`args.subparser_name == 'make_secure_bl1_image'` should be `args.subparser_name == 'secure-bootloader'` for `resocsec` if I understand correctly.